pages/delete_Accept_Button.tsx

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable react/jsx-no-comment-textnodes */
```

Comment:
this should be a component, can you move this to the components section and use it in your dashboard page? there should only be 1 page to represent the dashboard

Reply:
general tip is to reference the figma and if it's an entire figma rectangle screen to make it a page and if it's just an element within a screen to make it a component

Reply:
on another note, because the delete and accept buttons will be doing two separate things (delete will remove a user request on our users table and accept will change the user to admin approved), they should be 2 separate components